app/src/main/java/com/mealsmadeeasy/ui/home/suggestions/SuggestionClickListener.kt

```diff
@@ -0,0 +1,7 @@
+package com.mealsmadeeasy.ui.home.suggestions
+
+import android.view.View
+
+interface SuggestionClickListener {
```

Comment:
You don't actually have to declare functional interfaces in Kotlin. For example, `val listener: SuggestionClickListener = ...` is equivalent to `val listener: (View, String) -> Unit = ...`.